api/controllers/fda_applications.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const FdaApplication = require('../models/fda_application');
```

Comment:
Use `FDAApplication` instead, as `FDA` is an acronym (not only here, but throughout the PR)